### PR TITLE
OptimalPropeller function cleanup

### PR DESCRIPTION
**Pull request by aidanjungo**
## Proposed Changes

I used the OptimalPropeller function for another project a saw a lot of code that could be cleanup in order to be a bit more pythonic and easy to read.
 
## Related Work
- no

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [x] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

**Comment by pcarruscag** [comment]
Welcome to SU2 and thank you for your contribution. Is there something else you want to do in this PR? If not, please have a look at the CodeQL alert and then we can merge.

**Comment by aidanjungo** [comment]
Hello, sorry last week I did not had time to continue to work on it. If it is OK for you, I will make a few other changes between today and tomorrow, that could improve the code and then I think we could merge it.

**Comment by aidanjungo** [comment]
@pcarruscag could you run the workflow again, I made some small modifications, but now I cannot see the CodeQL alert anymore

File: SU2_PY/OptimalPropeller.py [hunk 1]
```diff
@@ -100,146 +119,110 @@
...
+rs_hub = rhub / R
 
 # Computation of the non-dimensional radial stations.
-for i in range(1,Stations+1):
-    r[i-1]=i/dStations
+for i in range(1, stations+1):
+    r[i-1] = i / float(stations)
     if r[i-1] <= rs_hub:
-        i_hub = i-1
+        i_hub = i - 1
```
[the remark this answers is not in the document]
Comment (aidanjungo) [reply SU2_PY/OptimalPropeller.py:170]: Thanks to this warning I found a small mistake I introduced during the refactor, it has been fixed by [a95cf4a](https://github.com/su2code/SU2/pull/1846/commits/a95cf4a326684b502683c646c10f6ed5d43bfd81). I checked with two different set of input values and it gives exactly the results file than the master branch.

**Review by WallyMaier — APPROVED** [review]
LGTM

**Review by pcarruscag — APPROVED** [review]
Add yourself to AUTHORS.md (in the root of the repo) please

**Pull request merged**